components/Nav/NavMobile/NavMobile.js

```diff
@@ -47,14 +47,14 @@ export default class NavMobile extends Component {
           </button>
         </Link>
 
-        <button
+        <button role='button' aria-label='Mobile Navigation Button'
```

Comment:
There's no need for `aria-label` here, since the button's purpose is described in plain text for screen readers (see line 56)